Chess/src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
+{
+	COORD topLeft = { 0, 0 };
+	HANDLE console = GetStdHandle(STD_OUTPUT_HANDLE);
+	CONSOLE_SCREEN_BUFFER_INFO screen;
+	DWORD written;
+
+	GetConsoleScreenBufferInfo(console, &screen);
+	FillConsoleOutputCharacterA(
+		console, ' ', screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	FillConsoleOutputAttribute(
+		console, FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_BLUE,
+		screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	SetConsoleCursorPosition(console, topLeft);
+}
+// create the GUI - ASCII art
+void Chess::setFrames() 
+{ 
+	// set all to ' ' (space bar ascii value 32) instead of 0  
+	{
+		for (size_t row = 0; row < _SIZE; ++row)
+			for (size_t col = 0; col < _SIZE; ++col)
+				m_board[row][col] = 32;
+	}
+	// set out frame
+	{
+		m_board[0][0] = 201;  m_board[0][20] = 187;
+		m_board[20][0] = 200; m_board[20][20] = 188;
+
+		for (size_t i = 1; i < 20; ++i)
+		{
+			m_board[0][i] = 205;
+			m_board[20][i] = 205;
+			m_board[i][0] = 186;
+			m_board[i][20] = 186;
+		}
+	} 
+
+	// set in side frame 
+	{
+		m_board[2][2] = 218;  m_board[2][18] = 191;
+		m_board[18][2] = 192; m_board[18][18] = 217;
+
+		for (size_t i = 4; i < 17; i += 2)
```

Comment:
Replace hardcoded values with constants or variables to make the code more flexible and easier to modify. For example, instead of repeatedly using 4 and 17 in multiple places, define constants for these values and reuse them.